go.mod

```
@@ -1,5 +1,5 @@
 module github.com/trussworks/terraform-aws-alb-web-containers
 
-go 1.13
+go 1.15
```

This can now be 1.16
```suggestion
go 1.16
```